From 298d312c0ff3f49061316ba799257277f4bb9bc4 Mon Sep 17 00:00:00 2001
From: Marek Szyprowski <m.szyprowski@samsung.com>
Date: Fri, 7 Aug 2026 12:23:34 +0200
Subject: [PATCH 12574/15333] MAINTAINERS: update tree for DMA MAPPING HELPERS

Update the git tree URL for DMA MAPPING HELPERS to point to the
maintainer's repository on kernel.org and remove the obsolete web
page reference to the old infradead.org tree.

Reported-by: Mostafa Saleh <smostafa@google.com>
Signed-off-by: Marek Szyprowski <m.szyprowski@samsung.com>
---
 MAINTAINERS | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..94330877ae72 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7664,8 +7664,7 @@ M:	Marek Szyprowski <m.szyprowski@samsung.com>
 R:	Robin Murphy <robin.murphy@arm.com>
 L:	iommu@lists.linux.dev
 S:	Supported
-W:	http://git.infradead.org/users/hch/dma-mapping.git
-T:	git git://git.infradead.org/users/hch/dma-mapping.git
+T:	git git://git.kernel.org/pub/scm/linux/kernel/git/mszyprowski/linux.git
 F:	include/asm-generic/dma-mapping.h
 F:	include/linux/dma-direct.h
 F:	include/linux/dma-map-ops.h
-- 
2.55.0

